clients/clearly-defined/src/main/kotlin/ClearlyDefinedService.kt

```diff
@@ -223,7 +223,7 @@ interface ClearlyDefinedService {
 
     /**
      * Get the curation for the component described by [type], [provider], [namespace] (use "-" if not applicable),
-     * [name] and [revision], see
+     * [name] and [revision] (when empty, the latest revision will be used if that makes sense for the provider), see
```

Comment:
I believe the docs should also state what the behavior is, when revision is empty and it doesn't make sense for the provider to query the latest revision.